doc/cephadm/install.rst

```diff
@@ -257,6 +257,42 @@ hosts to the cluster. No further steps are necessary.
     # ceph orch daemon add mon newhost1:10.1.2.123
     # ceph orch daemon add mon newhost2:10.1.2.0/24
 
+  .. note::
+     The **apply** command can be confusing. For this reason, we recommend using
+     YAML specifications. 
+
+     Each 'ceph orch apply mon' command supersedes the one before it, which means
+     that if you do not use the proper comma-separated list-based syntax when
+     you want to apply monitors to more than one host, you will clobber your
+     work as you go.
+
+     For example::
+        
+          # ceph orch apply mon host1
+          # ceph orch apply mon host2
+          # ceph orch apply mon host3
+
+     This results in only one host having a monitor applied to it: host 3.
+
+     To make certain that a monitor is applied to each of these three hosts,
+     run a command like this::
+       
+          # ceph orch apply mon "host1,host2,host3"
+
+     Instead of using the "ceph orch apply mon" commands, run a command like
+     this::
+
+          # ceph orch apply -i file.yaml
+
+     Here is a sample **file.yaml** file::
+
+          service_type: mon
+          placement:
+            hosts:
+             - host1
+             - host2
+             - host3
```

Comment:
Does it make sense to add MGRs here, too?
```suggestion
          service_type: mon
          placement:
            hosts:
             - host1
             - host2
             - host3
          ---
          service_type: mgr
          placement:
            hosts:
             - host1
             - host2
             - host3
```

just to show another example of how things work? I'd like to avoid people running into https://github.com/SUSE/doc-ses/issues/384 again.

Reply:
I ran into this too. Is it just the line separation that fixes this? Either way, we can probably document both? Leave Zac's original version, and then say "you can separate this into two files: mon.yml and mgr.yml if you want"

Reply:
To keep in mind also: https://github.com/SUSE/doc-ses/issues/391